core/templates/dev/head/components/ExplorationCreationService.js

```diff
@@ -18,11 +18,12 @@
  */
 
 oppia.factory('ExplorationCreationService', [
-  '$http', '$uibModal', '$timeout', '$rootScope', '$window',
+  '$http', '$rootScope', '$timeout', '$uibModal', '$window',
   'AlertsService', 'SiteAnalyticsService', 'UrlInterpolationService',
   function(
-      $http, $uibModal, $timeout, $rootScope, $window,
-      AlertsService, SiteAnalyticsService, UrlInterpolationService) {
+      $http, $rootScope, $timeout, $uibModal, $window,
+      AlertsService, SiteAnalyticsService, UrlInterpolationService
+  ) {
```

Comment:
Undo this change -- move the brackets to the previous line.

Reply:
Okay will do that !